src/server/pps/cmds/cmds.go

```diff
@@ -734,6 +741,11 @@ All jobs created by a pipeline will create commits in the pipeline's output repo
 			// validate flags
 			if raw && spec {
 				return fmt.Errorf("cannot set both --raw and --spec")
+			} else if !raw && !spec && output != "" {
+				fmt.Fprintln(os.Stderr, "-------------------------------")
+				fmt.Fprintf(os.Stderr, "output: %q\n", output)
+				fmt.Fprintln(os.Stderr, "-------------------------------")
```

Comment:
Are these first three messages really necessary?

Reply:
:man_facepalming: This is debugging code. I meant to remove this.

I'll check and see if I left anything else in, but thanks!